### ISSUE TITLE
 negligible levels of plasma NT- proANP

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#10432]
**Preferred term label:**

 negligible levels of plasma NT- proANP
**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
